conviva/src/main/java/com/bitmovin/analytics/conviva/BitmovinPlayerHelper.java

```diff
@@ -14,7 +13,6 @@ String getSdkVersionString() {
         return Player.getSdkVersion();
     }
 
-    @Nullable
```

Comment:
I added this in one of the previous commits, but didn't realise it's coming from the `androidx.annotation` package.